### PR TITLE
DEBUG variable in tools installation mode.

### PR DESCRIPTION
**Pull request by hascar**
Added check to set DEBUG variable in tools installation mode.
The tests are green here: https://jenkins.vpn.openquake.org/job/zdevel_oq-platform-tools/43/

### CODE REVIEW

File: openquake/server/local_settings.py.tools [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import sys
```
Comment (ptormene) [inline openquake/server/local_settings.py.tools:1]: Is `sys` needed too?

File: openquake/server/local_settings.py.tools [hunk 2]
```diff
@@ -24,4 +27,8 @@ WEBUI_ACCESS_LOG_DIR = '/var/log/oq-engine'
 # system time zone.
 # TIME_ZONE =
 
-DEBUG = False
+# Set debug if the installation is dev mode
+try:
```
Comment (ptormene) [inline openquake/server/local_settings.py.tools:31]: `DEBUG = os.environ.get('TOOLS_DEV', False) == 'True'`

Comment (hascar) [reply openquake/server/local_settings.py.tools]: Fixed with this commit https://github.com/gem/oq-engine/commit/ba295d939ba248f2a16d208b58debcc96228e435

File: openquake/server/local_settings.py.tools (hunk 1 above)
Comment (hascar) [reply openquake/server/local_settings.py.tools]: Fixed with this commit https://github.com/gem/oq-engine/commit/423d1b75e10d3132cfa75e4f2f46ad7908fe501c

**Review by ptormene — APPROVED** [review]

**Pull request merged**